build.gradle

```diff
@@ -47,13 +49,16 @@ apply from: 'version.gradle'
 ext {
     credentialsPropertyFile = 'credentials.properties'
     runsOnWindows = org.gradle.internal.os.OperatingSystem.current().isWindows()
+    spineProtobufPluginId = 'io.spine.tools.spine-model-compiler'
 
     projectsToPublish = ['datastore']
 }
 
 allprojects {
     apply plugin: 'maven'
     apply plugin: 'jacoco'
+    apply plugin: 'idea'
```

Comment:
I don't think we need that for the root project. So consider moving this to `subprojects`.